source/usage-examples/updateOne.txt

```diff
@@ -0,0 +1,32 @@
+=================
+Update a Document
+=================
+
+.. default-domain:: mongodb
+
+You can update a single document using the `updateOne() <https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#updateOne>`_ method. 
+``updateOne()`` takes a filter for a document, and updates the first document
+that matches the filter.
+
+Create an Options object to specify additional options. Specify the
```

Comment:
This line feels tautological. Could we use it to introduce the names of the options we're going to discuss to give it a bit more purpose?